src/examples/notifications/alert/AlertDefault.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import { Alert, Title, Close } from '@zendeskgarden/react-notifications';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const Example = () => (
+  <Row>
+    <Col>
```

Comment:
I don't see why Row-Col is needed here, unless we desire to constrain the width of the `Alert` (in which case, Col should receive a size).